client/src/Organisms/Nav/Nav.js

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
+import { useContext, useState } from 'react';
+
+import GoodBoiLogo from '../../Molecules/GoodBoiLogo/GoodBoiLogo';
+import NavButton from '../../Molecules/NavButton/NavButton';
+import NavButtonWrapper from '../../Molecules/NavButton/NavButtonWrapper';
+import NavButtons from '../../Molecules/NavButtons/NavButtons';
+import NavCornerMenu from '../../Molecules/NavCornerMenu/NavCornerMenu';
+import NavCornerMenuWrapper from '../../Molecules/NavCornerMenu/NavCornerMenuWrapper';
+import NavInnerStyled from './NavInnerStyled';
+import NavLinkStyled from '../../Atoms/NavLinkStyled/NavLinkStyled';
+import useClickOutside from '../../Hooks/useClickOutside';
+import { ReactComponent as AppLogoSVG } from '../../Assets/AppLogoSVG.svg';
+import { UserDataContext } from '../../Context/UserDataContext';
+
+const NavBackground = styled.header`
+  width: 100%;
+  height: 100%;
+  background: ${({ theme }) => theme.grey00};
+`;
+
+const Nav = () => {
+  const { state } = useContext(UserDataContext);
+  const { isAuthenticated } = state;
+
+  const [menuOpen, setMenuOpen] = useState(false);
+  const toggleHandler = () => {
+    setMenuOpen(!menuOpen);
+  };
+
+  let domNode = useClickOutside(() => {
+    setMenuOpen(false);
+  });
+
+  // when logged in:
+  if (isAuthenticated) {
```

Comment:
You could use ternany expression also to render one or another.

Reply:
It seems more readable for me with if statement, than looking for ":" in the code.